ocaml/testsuite/tests/typing-implicit-source-positions/application_via_option.ml

```diff
@@ -0,0 +1,119 @@
+(* TEST_BELOW
+Fille
+*)
+
+let f = fun ~(call_pos:[%call_pos]) () -> call_pos
+[%%expect {|
+val f : call_pos:[%call_pos] -> unit -> lexing_position = <fun>
+|}]
+
+let _ = f ?call_pos:None ();
+[%%expect {|
+Line 1, characters 20-24:
+1 | let _ = f ?call_pos:None ();
+                        ^^^^
+Error: the [%call_pos] label 'call_pos' is not optional. It
+       cannot be applied with '?'.
```

Comment:
I find this phrasing a little confusing. Might I suggest

```suggestion
Error: the argument labeled 'call_pos' is a [%call_pos] argument, filled in
       automatically if omitted. It cannot be passed with '?'.
```

Reply:
I like it! I've changed the error message + accepted this corrected output.